schema/service-spec.json

```diff
@@ -16,6 +16,10 @@
           "type": "string",
           "default": "Dockerfile"
         },
+	"target": {
+          "description": "specify the target in case of multi-stage dockerfile. Docker will start building the image from the specified taget",
```

Comment:
1. Please correct the description, "target" would be used to specify till what stage Dockerfile has to be built. 
2. typo